src/libostree/ostree-repo.h

```diff
@@ -735,7 +735,7 @@ ostree_repo_checkout_tree (OstreeRepo               *self,
                            GError                  **error);
 
 /**
- * OstreeRepoCheckoutOptions:
+ * OstreeRepoCheckoutOptions: (skip)
```

Comment:
Hmm...I have a worry that someone could have actually been using this *without* the bitfield parts.  But OK.